README.md

```diff
@@ -69,7 +69,13 @@ there are two ways to specify the host in the jolokia URL so this URL will be re
 10. If you wish to run this as a service you need to create a service wrapper for it. Any pull requests for making it are welcome! If it's possible running it as docker making it simpler.
 
    
-
+## As Java Agent
+This lib can also get the metrics from MBean Platform instead of jolokia. In order to do so, we need to run inside the JVM.
+- First, get the java agent jar from the releases page
```

Comment:
Maven Central is not good for standalone Jars..
Like jolokia or others stand alones, we can just publish the JAR.

Its still not there, but when the PR is done i can just add the jar